### PR TITLE
Init commit + complete feature for this new branch

### PR DESCRIPTION
**Pull request by ekhaemba**

### CODE REVIEW

**Comment by robbawebba** [comment]
Sweet, that was quick!

File: src/client/app/Reducers/user_reducer.js [hunk 1]
```diff
@@ -1,34 +1,82 @@
...
   switch (action.type) {
     case LOGIN_USER:
-      return Object.assign({},state,
-        {
-          username : action.username,
-          full_name : action.full_name,
-          logged_in : true
-        });
+      return {
+        ...state,
```
Comment (robbawebba) [inline src/client/app/Reducers/user_reducer.js]: I'm getting a compile error here

**Comment by robbawebba** [comment]
@ekhaemba @reitersg When testing this code, did you guys get an error about the spread operator in user_reducer.js [here](https://github.com/ekhaemba/scheduling-application-utilizing-react-redux/pull/10/files#diff-277ccd1d2cd66d49d46f0dcaa280cbf3R14)?

**Comment by robbawebba** [comment]
To fix it we needed to require the `babel-preset-stage-2` plugin for babel. I'll create a new PR for it.

**Comment by ekhaemba** [comment]
Oh, my bad I forgot to include my babelrc in my commit.
```
{
  "presets" : ["es2015", "react"],
  "plugins": ["transform-object-rest-spread"]
}
```

**Pull request merged**